src/go/k8s/pkg/resources/configmap.go

```diff
@@ -141,6 +147,17 @@ func (r *ConfigMapResource) createConfiguration() *config.Config {
 		}
 	}
 
+	replicas := *r.pandaCluster.Spec.Replicas
```

Comment:
so with this, all nodes will get all nodes as seed servers, is that what we want? FroM @BenPope 's comment I understood it should be:
node 0: node 0
node 1: node 0, 1
node 2: node 0, 1, 2

If we give 
node 0: node 0, 1, 2

will this work even if the nodes does not exist yet? and if we're creating them and not doing rolling update?

Reply:
As @BenPope said. 

> node 0: node 0

The configuration will remove seed servers.

> node 1: node 0, 1

Will have all nodes. The process of creating ring 0 raft group will start from first on the list (`node-0`). The none initialised node (without ring 0 raft group) will refuse to respond to join request. If we lost `data` folder, then recover process should reach any node in the working cluster. The node 0 is problematic. 

 > will this work even if the nodes does not exist yet?

Yes as the retries mechanism is build in the joining process.

> if we're creating them and not doing rolling update?

Sorry, I don't understand. Could you rephrased the question?